packages/legend-graph/src/graph-manager/protocol/pure/v1/V1_PureGraphManager.ts

```diff
@@ -3153,33 +3133,19 @@ export class V1_PureGraphManager extends AbstractPureGraphManager {
         const graphData = this.getFullGraphModelData(graph);
 
         services.forEach((service) => {
-          if (
-            this.TEMPORARY__enableNewServiceRegistrationInputCollectorMechanism
-          ) {
-            const pmcd = new V1_PureModelContextData();
-            pmcd.serializer = graphData.serializer;
-            pmcd.INTERNAL__rawDependencyEntities =
-              graphData.INTERNAL__rawDependencyEntities;
-            pmcd.elements = graphData.elements;
-            const sdlcInfo = new V1_LegendSDLC(groupId, artifactId, version);
-            sdlcInfo.packageableElementPointers = [
-              new V1_PackageableElementPointer(
-                PackageableElementPointerType.SERVICE,
-                service.path,
-              ),
-            ];
-            inputs.push({ service: service, context: pmcd });
-          } else {
-            const prunedGraphData =
-              this.TEMPORARY__pruneServicesFromGraphForRegistration(
```

Comment:
we need to remove `TEMPORARY__pruneServicesFromGraphForRegistration` and its associates as well